install.sh

```diff
@@ -492,6 +797,19 @@ get_cwd_git_dir() {
     return 0
 }
 
+############################################################
+# Gets all worktrees attached to the given repository `$1`
+#   and sets `REPO_WORKTREES`
+#
+# Returns: None
+############################################################
+get_repo_worktrees() {
+    # This feature is kind of buggy in earlier version of git < 2.28.0
```

Comment:
Does it work in `< 2.28.0`? (with that caveat)
So this is still compatible with older `git` versions, right?

Reply:
yes